### ISSUE TITLE
Lengths, angles, etc can only be modified in the code editor

### ISSUE THREAD

**Issue by lf94** [KittyCAD/modeling-app#2667]
This is a less severe issue because in a sense this is equivalent to the standard side menus MEs deal with.

If we want to go this route, let's maybe make variables more glaringly obvious ?

**Comment by franknoirot** [comment]
Are we talking sketch, modeling operations like Extrude, or both? It might help to split those two apart for scope. At the very least I'll try to address both in my UX recommendations.

**Comment by lf94** [comment]
Yep, pretty much all of it. Constraint asking for length, extrude asking for length, etc.

**Comment by franknoirot** [comment]
On the sketch front, #2692 and #2693 will help considerably. It will be more difficult, and needs a separate issue, to outline the behavior for going back and modifying the length, because it may no longer be editable because it is dependent on a constraint in some way.

**Comment by lf94** [comment]
Oh nice! Yeah when those two issues are resolved we can resolve this one I suppose. If we make it so when a length or angle is hovered over, and the respective variable / value is highlighted, that'd be enough I think.
